test/meetup.spec.js

```diff
@@ -21,8 +21,8 @@ describe("/Meetups Resources", () => {
           expect(res.type).to.eql("application/json");
           expect(res.body.status).to.equal(201);
```

Comment:
What is the difference between `.eql ` and `.equal`, find out if there is a difference if none, just use one of the two

Reply:
GM I refactored the test to use equal thanks for the review